pkg/core/label_list_test.go

```diff
@@ -157,3 +157,7 @@ func TestListLabels(t *testing.T) {
 		}
 	}
 }
+
+func TestListLabelVersions(t *testing.T) {
+	t.Skip("tbd")
```

Comment:
ok. So here is a todo, isn't it?

Reply:
this is the tail end of the wip here, yep